python/lsst/ap/association/trailedSourceFilter.py

```diff
@@ -115,11 +118,12 @@ def _check_dia_source_trail(self, dia_sources, exposure_time, flags):
         -------
         trail_mask : `pandas.DataFrame`
             Boolean mask for DIASources which are greater than the
-            cutoff length and have the edge flag set.
+            cutoff length or have off_image setl. Also checks if both
+            suspect_long_trail and edge are set and masks those sources out.
         """
-        trail_mask = (dia_sources.loc[:, "trailLength"].values[:]
-                      >= (self.config.max_trail_length*exposure_time))
-
-        trail_mask[np.where(flags['ext_trailedSources_Naive_flag_edge'])] = True
+        trail_mask = dia_sources.loc[:, "trailLength"] >= (self.config.max_trail_length*exposure_time)
+        trail_mask |= flags['ext_trailedSources_Naive_flag_off_image']
+        trail_mask |= (flags['ext_trailedSources_Naive_flag_suspect_long_trail']
```

Comment:
I think it would look about the same if you formatted it one per line, but not a huge deal. Example
```
trail_mask = complicated_thing \
           |= other_complicated_thing \
           |= yet_another_very_long_thing_hooray
```